pkg/v1/resource/chartconfig/desired.go

```diff
@@ -4,68 +4,30 @@ import (
 	"context"
 
 	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
-	"github.com/giantswarm/microerror"
-	apimetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
-	"github.com/giantswarm/cluster-operator/pkg/cluster"
-	"github.com/giantswarm/cluster-operator/pkg/label"
 )
 
 const (
 	chartConfigAPIVersion           = "core.giantswarm.io"
 	chartConfigKind                 = "ChartConfig"
 	chartConfigVersionBundleVersion = "0.1.0"
-
-	chartOperatorChart           = "quay.io/giantswarm/chart-operator-chart"
-	chartOperatorChartConfigName = "chart-operator"
-	chartOperatorChannel         = "stable"
-	chartOperatorRelease         = "chart-operator"
 )
 
 // GetDesiredState returns all desired ChartConfigs for managed guest resources.
 func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interface{}, error) {
-	clusterGuestConfig, err := r.toClusterGuestConfigFunc(obj)
-	if err != nil {
-		return nil, microerror.Mask(err)
-	}
-
-	clusterConfig, err := prepareClusterConfig(r.baseClusterConfig, clusterGuestConfig)
-	if err != nil {
-		return nil, microerror.Mask(err)
-	}
+	// TODO: Add ChartConfigs as we migrate components from k8scloudconfig.
+	/*
+		clusterGuestConfig, err := r.toClusterGuestConfigFunc(obj)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+
+		clusterConfig, err := prepareClusterConfig(r.baseClusterConfig, clusterGuestConfig)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+	*/
```

Comment:
Removing the chart-operator-chart from the desired chartconfig resources since it will be installed via the chart resource. We will add the guest cluster components here as we migrate them from k8scloudconfig.